tests/unit/lms/services/vitalsource/factory_test.py

```diff
@@ -6,11 +6,12 @@
 from lms.services.vitalsource.factory import service_factory
 
 
+@pytest.mark.usefixtures("application_instance_service")
 class TestServiceFactory:
```

Comment:
I really hate these tests. I'm coming to the conclusion that it's easier to test 100 lines of code with no-ifs than 5 lines of code with 5 ifs.